src/main/java/bit/javaoop/headquartersPackage/CEO.java

```diff
@@ -0,0 +1,23 @@
+package bit.javaoop.headquartersPackage;
+
+import bit.javaoop.managersPackage.Manager;
+
+import java.util.ArrayList;
+
+public class CEO {
+    private String name;
+
+    public CEO(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    private static ArrayList<Manager> managersList;
```

Comment:
- To tak samo powinno być na samej górze generalnie zmienne/stałe deklarujemy na samej górze potem konstruktory i dopiero metody, poza tym to nie powinien być static:) tylko zwykła prywatna zmienna inicjalizowana w konstruktorze.
- Oprócz tego, powinniśmy dodać getter na tą listę i w nim zwracać jej kopię żeby nam nikt jej nie zepsuł, ale żebyśmy potem mogli dostać się do niej w klasie Company kiedy będziemy chcieli przeiterować się po wszystkich managerach i pracownikach i ich wypisać
![image](https://user-images.githubusercontent.com/18233796/38770933-7ed7518c-401a-11e8-962a-dbd5a989a634.png)